fmn/testing/message.py

```diff
@@ -1,9 +1,6 @@
 import pytest
 from fedora_messaging import message
 
```

Comment:
What's the reason to move this from `tests/` into the `fmn` package (where it will be installed)? We can have importable modules inside `tests/`…

Reply:
Ah, how do you do that? I tried to put the message class in a module at the top of the `tests` tree, but `from ...utils import Message` failed because it could not find the module.

Reply:
To be precise I get: `ImportError: attempted relative import beyond top-level package`

Reply:
OK, in a followup commit I've changed how mock messages are created, and it gets rid of the `fmn.testing` module.

I'll squash when the PR is approved.